miniscope_io/stream_daq.py

```diff
@@ -496,6 +512,17 @@ def capture(
         else:
             raise ValueError(f"source can be one of uart or fpga. Got {source}")
 
+        # Video output settings
+        fourcc = cv2.VideoWriter_fourcc(*'DIVX')
+        frame_rate = 20.0 # this should be in the config yaml
+        frame_size = (self.config.frame_width, self.config.frame_height)
+        out = cv2.VideoWriter('output.avi', fourcc, frame_rate, frame_size)
```

Comment:
see this method here - let's make a generalized `VideoWriter` wrapper class and use it throughout the package for consistent behavior. https://github.com/Aharoni-Lab/miniscope-io/blob/aabd2322af5f60fb91693679a0d4188a6606a771/miniscope_io/io.py#L362